src/corelibs/U2Core/src/globals/GUrl.h

```diff
@@ -78,7 +78,7 @@ class U2CORE_EXPORT GUrl {
     // The function converts url string to multibyte form
     // default code page is CP_THREAD_ACP
     // must use "delete" to delete returned value
```

Comment:
Remove this comment line: it is wrong now.

Reply:
done